Helper/DeviceView.php

```diff
@@ -66,12 +63,20 @@ class DeviceView
     protected $switchParam = self::SWITCH_PARAM_DEFAULT;
 
     /**
+     * @var array
+     */
+    protected $configRedirect;
+
+    /**
      * Constructor
      *
-     * @param \Symfony\Component\DependencyInjection\Container $serviceContainer
+     * @param array        $configRedirect
+     * @param RequestStack $requestStack
      */
-    public function __construct(RequestStack $requestStack = null)
+    public function __construct($configRedirect = null, RequestStack $requestStack = null)
```

Comment:
Adding a new argument to the beginning of the parameters list is a BC break. To ease migration I would add it after existing argument.